src/explore-education-statistics-admin/src/pages/release/content/ReleaseContentPage.tsx

```diff
@@ -88,7 +102,37 @@ const ReleaseContentPageLoaded = () => {
                     </h2>
 
                     <ReleaseContentHubContextProvider releaseId={release.id}>
-                      <ReleaseContent />
+                      <ReleaseContent
+                        transformFeaturedTableLinks={
+                          canPreviewRelease
+                            ? (url: string, text: string) => {
+                                return (
+                                  <ButtonText
+                                    onClick={() => {
+                                      const dataBlockParentId = url
```

Comment:
Might not be immediately obvious what part of the URL we're trying to extract here, as we might not know what the "?" denotes unless stepping through the application.  I'm expecting we're expecting "?featuredTable".

Maybe a code comment, or use a regex with a capture group to grab the parent id, or specifically mention what query parameter you're expecting to find at the end of the URL e.g. .split('?featuredTable')[0]?

Reply:
I've added a comment now.

Reply:
Cool ta